include/swift/Serialization/SerializedSILLoader.h

```diff
@@ -112,6 +112,10 @@ class SerializedSILLoader {
 
   /// Deserialize all SILFunctions, VTables, and WitnessTables for
   /// a given Module.
+  void getAllForModule(Identifier Mod);
```

Comment:
No it's the opposite. If you pass a null primary file below, we call getAll(true) on each iteration. I want to call getAll(false).